doc/pull_request_template.md

```diff
@@ -0,0 +1,19 @@
+### Description
+
+Write in as much detail as possible on the problem that this PR solves or the feature that it adds.
+This will give the reviewers more context on this pull request.
+
+### Screenshots
+
+Attach screenshots or diagrams, if applicable.
+
+### Developer checklist
+- [ ] If there is a Jira task, c/p the task description to this PR
+- [ ] Assign yourself to this pull request
+- [ ] Open a DRAFT pull request if it's not ready for review yet
+- [ ] Tag the reviewers for review
+- [ ] Attach appropriate labels to the PR (`enhancement`, `bug`, `work-in-progress`, etc.)
+- [ ] Don't force-push any changes after a review has been done by any of the reviewers, unless agreed upon with the reviewer.
```

Comment:
Is this checklist item or just note?

Reply:
Yeah just a note. We can add a separate section for notes.